server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -101,7 +102,19 @@
   private Long maximumSecretSizeInBytesInclusive;
 
   public enum RowHmacCheck {
-    DISABLED, DISABLED_BUT_LOG, ENFORCED
+    DISABLED("disabled"),
+    DISABLED_BUT_LOG("logging"),
+    ENFORCED("enforced");
+
+    private final String configEntry;
+
+    RowHmacCheck(String configEntry) {
+      this.configEntry = configEntry;
+    }
+
+    public String getConfigEntry() {
+      return configEntry;
+    }
   }
```

Comment:
For testing, we need a way to override the configured HMAC check configuration so that mismatches fail. Since enum-to-string translation is now two-way I decided to store the config file tokens directly with the enums themselves in order to simplify things and increase maintainability.